.github/workflows/ci.yml

```diff
@@ -128,13 +132,29 @@ jobs:
             --output appimage && \
           mv *.AppImage dist/
 
+      - name: Package (3DS)
+        if: ${{ matrix.n3ds }}
+        run: |
+          cd build
+          mkdir dist
+          mv *.3dsx dist/
+
       - name: Upload Build Artifacts
+        if: ${{ !matrix.n3ds }}
         uses: actions/upload-artifact@v4
         with:
           name: '${{ matrix.name }}'
           path: |
             build/dist/isle-*
             build/dist/*.AppImage
+            build/dist/*.3dsx
+
+      - name: Upload 3dsx
```

Comment:
This step is not needed. The upload happens in the previous step.

Reply:
Right now, you're creating a `isle.3dsx.zip` archive. Is that more useful?

Reply:
Where? I see the `isle.3dsx` on the CI summary (unless I'm seeing it wrong because macOS is automatically unzipping it?)

Reply:
OH you're right. whoops.